src/components/Moveable.vue

```diff
@@ -28,6 +28,35 @@ const ALLOWED_EVENTS = [
   'pinchEnd',
 ];
 
+const MOVEABLE_PROPS = [
+  'draggable',
+  'keepRatio',
+  'origin',
+  'pinchable',
+  'resizable',
+  'rotatable',
+  'scalable',
+  'throttleDrag',
+  'throttleResize',
+  'throttleScale',
+  'throttleRotate',
+  'warpable',
+];
+
+const watchReactiveProp = (key, deep) => ({
+  handler(newValue) {
+    const existingValue = this.moveable[key];
+    if (existingValue === newValue) return;
+    this.moveable[key] = newValue;
+  },
+  deep,
+});
+
+const watchMoveableProps = () => MOVEABLE_PROPS.reduce((acc, prop) => {
+  acc[prop] = watchReactiveProp(prop, true);
```

Comment:
I don't think we need `deep` here 🤔 
But let it be.

Reply:
Absolutely agree, I decided to implement it in this way to accommodate for future upgrade to newer versions of `Movable`, so we don't have to think about changes in underlying API.